packages/ui/src/hooks/use-page-context.ts

```diff
@@ -24,11 +26,13 @@ const noop = () => {
 
 export const PageContext = createContext<Context>({
   toast: '',
+  theme: 'light',
```

Comment:
Default theme should be `system`?

Reply:
OK, just scrolled down and realized there is no `system` option in UI. Is this by design?

Reply:
slightly different implementation in the use-theme hook.  Here in context theme just represent the currently active mode value light or dark, after since with the system.

Reply:
ok